frontend/src/components/resource/resources.css

```diff
@@ -117,20 +126,38 @@
   justify-content: space-around;
   align-items: center;
 }
-
+.form-row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+}
+.form-group2 {
+  padding-left: 1rem;
+}
 .resource-header-text {
+  padding: 30px 0;
   font-size: 30px;
+  text-align: center;
+  color: var(--secondary-color);
   text-transform: capitalize;
-  color: #121f3a;
-  text-align: center !important;
+}
+.resource-card {
```

Comment:
Add padding to the card. There isn't any space between the submit button and the end of the container